dashboard/test/ui/features/applab/dataBlocks.feature

```diff
@@ -1,13 +1,12 @@
 @no_ie9
 @dashboard_db_access
+@as_student
 Feature: App Lab Data Blocks
```

Comment:
To my knowledge you can't tag a scenario background (at least, I can't find any examples of it online) and I think tagging the scenario achieves what you're talking about here.  In this case I'm also concerned that adding `@as_student` to the background reads like you _only_ need to be a student for the background portion.

In my mind, this isn't so different from the `@dashboard_db_access` tag, in that it specifies a declarative environmental prerequisite for the Feature/Scenario, rather than an "action."  If you have a test that actually needs to check the sign-in/out steps you wouldn't use this tag.